Scripts/projects.js

```diff
@@ -0,0 +1,11 @@
+(function(module) {
+  var projectController = {};
+
+  // Done: Define a function that hides all main section elements, and then reveals just the #about section:
+  projectController.index = function() {
+    $('.swapped-body-content').hide();
+    $('.projects').show();
+  };
+  console.log("hi");
+  module.projectController = projectController;
+})(window);
```

Comment:
It looks like this file is mostly a duplicate of projectController.js, and is not referenced anywhere. It should be removed if  it's not being used.